### ISSUE TITLE
authorが「制作者」と訳されている箇所がある

### ISSUE THREAD

**Issue by momdo** [waic/wcag22#302]
例えば、用語のコントラスト比 注記6

https://waic.jp/translations/WCAG22/#h-note-96

> WCAG に適合しているか判断する場合は、典型的な提示において隣接するであろうと<mark>制作者</mark>が想定するコンテンツで指定された色の組み合わせについて評価しなければならない。<mark>制作者</mark>は、ユーザエージェントによる色の変更などのように通常とは異なる提示を考慮する必要はない。ただし、それが<mark>制作者</mark>のコードによって起こる場合は除く。

https://www.w3.org/TR/2023/REC-WCAG22-20231005/#h-note-96

> WCAG conformance should be evaluated for color pairs specified in the content that an <mark>author</mark> would expect to appear adjacent in typical presentation. <mark>Authors</mark> need not consider unusual presentations, such as color changes made by the user agent, except where caused by <mark>authors'</mark> code.

他の箇所にもあるかどうかを確認しておく

**Comment by momdo** [comment]
5.2.2 ウェブページ全体

数が合わない？

https://www.w3.org/TR/2024/REC-WCAG22-20241212/#cc2
> Note 2
<mark>Authors</mark> of web pages that cannot conform due to content outside of the <mark>author's</mark> control may consider a Statement of Partial Conformance.

https://waic.jp/translations/WCAG22/#cc2
>注記 2
<mark>コンテンツ制作者</mark>が制御できないコンテンツがあるために適合できないウェブページについては、部分適合に関する記述を行うことを検討するとよい。

**Comment by momdo** [comment]
「サイズの大きな (テキスト)」は、https://github.com/waic/wcag22/pull/1412 で対応。

**Comment by caztcha** [comment]
PR 作成します。
